docs/pelion_ready/device-management-config.md

```diff
@@ -132,9 +140,9 @@ After you've successfully passed the "Connect" tests as described above, you can
    ```
 
 <span class="notes">**Note:** `mbed-bootloader` is primarily optimized for `GCC_ARM`, so you may want to compile it with that toolchain.
-Before jumping to the next step, you should compile and flash the bootloader and then connect over the virtual comport to ensure the bootloader is running correctly. You can ignore errors related to checksum verification or failure to jump to application - these are expected at this stage.</span>
+Before adding the bootloader to your application, you need to compile and flash the bootloader and then connect over the virtual com port to ensure the bootloader is running correctly. You can ignore errors related to checksum verification or failure to jump to application - these are expected at this stage.</span>
 
-## 3. Add the bootloader to your application
+## Adding the bootloader to your application
```

Comment:
Do these steps need to happen in order? If so, is there a way to indicate that without numbering the headings?

Reply:
Numbering in headings is a bit questionable.

Reply:
Heading levels, perhaps, but less noticeable in the rendered doc than the raw markdown.

Reply:
Manual numbering goes wrong very quickly. I've asked for automated numbering to be added to the CSS, but we don't have it yet.